streams/src/test/java/org/apache/kafka/streams/perf/SimpleBenchmark.java

```diff
@@ -101,9 +104,9 @@
         }
     };
 
-    private static int numRecords;
-    private static int processedRecords = 0;
-    private static long processedBytes = 0;
+    public static int numRecords;
```

Comment:
Also, it might be better to not have this as static. It is a field associated with a particular instance of the benchmark. Maybe pass the value in as a ctor param?